iree/hal/vulkan/vma_allocator.cc

```diff
@@ -23,6 +22,7 @@
 #include "iree/hal/vulkan/vma_buffer.h"
 
 #if VMA_RECORDING_ENABLED
```

Comment:
(as you mention) it'd be good follow-up to make this options on the creation instead so we can remove this (and compile out the options in release builds/etc)

Reply:
Done in this PR. PTAL.